src/mixins/filing-template-mixin.ts

```diff
@@ -289,10 +289,19 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
           legalName: this.getBusinessLegalName
         },
         dissolution: {
+          dissolutionDate: this.getCurrentDate,
           affidavitConfirmed: this.getAffidavitStep.validationDetail.validationItemDetails[0]?.valid || false,
           custodialOffice: this.getBusiness.officeAddress,
           dissolutionType: this.getDissolutionType,
-          parties: [this.getCustodian]
+          parties: [{
+            ...this.getCustodian,
+            roles: [
+              {
+                roleType: RoleTypes.CUSTODIAN,
+                appointmentDate: this.getCurrentDate
```

Comment:
I just checked. For a COD filing, this is YYYY-MM-DD Pacific.

https://github.com/bcgov/business-filings-ui/blob/a3ecbcbca80c5421cb4dacad3a33999c047583c7/src/components/common/Directors.vue#L891

Reply:
The format of `getCurrentDate` is also YYYY-MM-DD or is there is a separate concern here?

Reply:
I just wanted to confirm that the appointment date is the same as in the COD filing.

It's confirmed. No further action needed.

Reply:
ahh i see, okie doke!

Reply:
You keep using that word. I do not think it means what you think it means.

https://www.urbandictionary.com/define.php?term=Okie-Doke

Reply:
😆 
I know! We have discussed this haha

I first heard this from "Ned Flanders" on the Simpsons... i wonder if Matt Groening knew this when writing those episodes 😁

Reply:
FWIW, the Great Vizzini kept using his word, too 😁

Reply:
hahah who is that? I had to google.
Is this a Princess Bride reference ??